content/en/docs/Tasks/creating-an-update-graph.md

```diff
@@ -116,6 +120,43 @@ The CSV name currently installed must be in the `skips` field of a CSV in the ne
 
 The version currently installed must be in the `olm.skipRange` field of a CSV in the new channel.
 
+
+`replaces and skips example:`
+
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: etcdoperator.v0.9.2
+  namespace: placeholder
+  annotations:
+spec:
+    displayName: etcd
+    description: Etcd Operator
+    replaces: etcdoperator.v0.9.0
+    skips:
+    - etcdoperator.v0.9.1
+```
+
+The `replaces` field contains the current installed CSV. `Skips` parameter contains the version that needs to skip during the upgrade process. There might be some operator versions that have the bug. A `skips` parameter is used to ignore the faulty upgrade of the operator.
+
+#### Exmple for SkipRange
+
+`skipRange example:`
+
+```yaml
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+    name: elasticsearch-operator.v4.1.2
+    namespace: placeholder
+    annotations:
+        olm.skipRange: '>=4.1.0 <4.1.2'
+```
+
+The skipRange parameter is used to ignore the multiple versions of an operator during the upgrade cycle. In the above example the operator version will be avoided from 4.1.0 to 4.1.1.
```

Comment:
we need to add here that is not recommended outbound ranges such as `<4.1.2` 
also see that values requires to be in "`"

Reply:
Added values in "`"